modules/irsa/variables.tf

```diff
@@ -88,13 +88,38 @@ variable "additional_irsa_configs" {
   }
 }
 
```

Comment:
if `kms_info` `region` are needed by `external_deployments_operator` they should be fields and not required variables by the module